### ISSUE TITLE
Any update?

### ISSUE THREAD

**Issue by ale5000-git** [mikereidis/spirit2_free#17]
@mikereidis: Could you please update it?

The latest version is really old.
Thanks in advance.

**Comment by mikereidis** [comment]
Mike passed away. No update available. Maybe the paid version was more
updated.
His wife

**Comment by ale5000-git** [comment]
I'm sorry for him.

PS: Isn't there anyone else working on this project?

**Comment by mirh** [comment]
Lucie, is there any chance for you to upload source code of his works somewhere?
It's not an understatement to say there wasn't a more knowledgeful and capable person on FM matters worldwide.

**Comment by mikereidis** [comment]
Not that I know.

**Comment by mikereidis** [comment]
No. He passed away to quickly. I have a diabetic autistic boy to take care
of. Mike worked a lot for not much money and not many people are ready to
do that.
His wife

**Comment by mirh** [comment]
Source code would allow people to improve (and keep working further) the program. 
Then, if it's just about money, I don't want to claim anything - but for as much as I know, a lot of developers would be happy to create a *thank-you* "pro" tier with proceeds to go back to you. 

But if you don't have access to it (disks were thrown away, or something), then I can only just offer you again my condolences and best luck.

**Comment by gohanko** [comment]
Just fork the project and work on it. Don't be inconsiderate.

This is the first time I heard of a developer of an app I'm using passed away, weird feeling.

**Comment by ThinkDigitalSoftware** [comment]
No one is being inconsiderate here, in fact, it's very considerate. We appreciate the work and our condolences 💔

**Comment by paolo-caroni** [comment]
> Just fork the project and work on it.

About this, there is [RFM](https://github.com/vladislav805/RFM-Radio) that is "inspired" by Spirit 1 and 2.

Also I will use this message to give my condolences to the family and friends of Mike Raid.
